tests/micro/hellojob.py

```diff
@@ -0,0 +1,19 @@
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class VSCJobTest(rfm.RunOnlyRegressionTest):
+    descr = "test running job"
+    valid_systems = ["*"]
```

Comment:
This test should only be run inside a job, without parallelism, so I would say `valid_systems = ["*:single-node"]` (additionally I would rename `single-node` to `serial` in the settings file and thus here as well).

Reply:
ok! read everything, I merged the mpi in my local fork as well, working

Reply:
don't know if the change of the naming convention is appropriate. probably is better to discuss it the 10th. For the moment let's stick with the more verbose single-node